neo/api/JSONRPC/JsonRpcApi.py

```diff
@@ -196,8 +168,8 @@ def json_rpc_method_handler(self, method, params):
             return len(NodeLeader.Instance().Peers)
 
         elif method == "getcontractstate":
-            script_hash = self.parse_uint_str(params[0])
-            contract = Blockchain.Default().GetContract(script_hash)
+            script_hash = UInt160.ParseString(params[0])
+            contract = Blockchain.Default().GetContract(script_hash.ToBytes())
             if contract is None:
                 raise JsonRpcError(-100, "Unknown contract")
```

Comment:
Sorry one last item (was waiting for the new fixtures to download). There's no test covering this exception branch. Would be great to have that, because currently we're near 100% coverage (just need to add the unimplemented methods and cover this path) 💯